cypress/support/index.js

```diff
@@ -2,7 +2,7 @@ import './commands';
 
 Cypress.on('uncaught:exception', (err, runnable, promise) => {
   console.log(err.payload.response.status);
-  if (promise && err.payload && err.payload.response.status == 404) {
+  if (promise && err.payload && err.payload.response.status === 404) {
```

Comment:
Are we sure this still works? If status is a string this change will break the comparison

Reply:
Not really, I just found it weird that `yarn flow` reported it so Thought maybe to change it 🤷 
I'll change it back and if it comes up again we can do more research :)